packages/ui/src/composites/wui-list-transaction/index.ts

```diff
@@ -0,0 +1,49 @@
+import { html, LitElement } from 'lit'
+import { customElement, property } from 'lit/decorators.js'
+import '../../components/wui-text'
+import '../wui-transaction-visual'
+import { elementStyles, resetStyles } from '../../utils/ThemeUtil'
+import styles from './styles'
+import type { TransactionType } from '../../utils/TypesUtil'
+
+@customElement('wui-list-transaction')
+export class WuiListTransaction extends LitElement {
+  public static styles = [resetStyles, elementStyles, styles]
+
+  // -- State & Properties -------------------------------- //
+  @property() public type: TransactionType = 'bought'
+
+  @property({ type: Boolean }) public disabled = false
+
+  @property() public imageSrc = ''
+
+  @property() public date = ''
+
+  @property() public transactionDetail = ''
+
```

Comment:
Date should be of `Date` type and we should format it via https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat